tests/sentry/backup/test_correctness.py

```diff
@@ -24,24 +46,76 @@ def import_export_then_validate(tmp_path: Path, fixture_file_name: str) -> None:
         rv = CliRunner().invoke(import_, [str(fixture_file_path)])
         assert rv.exit_code == 0, rv.output
 
-    res = validate(expect, tmp_export_to_file(tmp_path.joinpath("tmp_test_file.json")))
+    res = validate(
+        expect,
+        tmp_export_to_file(tmp_path.joinpath("tmp_test_file.json")),
+        map,
+    )
     if res.findings:
         raise ValidationError(res)
 
 
 @django_db_all(transaction=True, reset_sequences=True)
-@freeze_time("2023-06-22T23:00:00.123Z")
 def test_good_fresh_install_validation(tmp_path):
-    import_export_then_validate(tmp_path, "fresh-install.json")
+    import_export_then_validate(tmp_path, "fresh-install.json", DEFAULT_COMPARATORS)
 
 
 @django_db_all(transaction=True, reset_sequences=True)
 def test_bad_fresh_install_validation(tmp_path):
+
     with pytest.raises(ValidationError) as excinfo:
-        import_export_then_validate(tmp_path, "fresh-install.json")
-    assert len(excinfo.value.info.findings) == 2
+        import_export_then_validate(tmp_path, "fresh-install.json", {})
```

Comment:
Looks goofy to me to pass `{}` as the final arg everywhere. Give the arg a default? But in that case it feels like `DEFAULT_COMPARATORS` should be the default. 🤔

Reply:
I took the default suggestion. I think keeping it `DEFAULT_COMPARATORS` is fine: the `DEFAULT...` refers to production uses, I think it's to be expected that tests might override a production default to validate behavior.